### ISSUE TITLE
Build GitHub action to replace doctr

### ISSUE THREAD

**Issue by mcg1969** [conda/conda-pack#126]
With the move to GitHub actions the one thing we postponed was an update to the github pages pusher. We'll need a new deploy key most likely.

**Comment by mcg1969** [comment]
This was the Travis approach. During the build:
```
cd docs
conda install -y --use-local sphinx numpydoc doctr sphinxcontrib-autoprogram make conda-pack
make html || exit -1
```
And in the "after success" step:
```
doctr deploy . --built-docs build/html/
```

**Comment by mcg1969** [comment]
This is likely the GitHub action to use: https://github.com/marketplace/actions/github-pages-action

**Comment by mcg1969** [comment]
Fixed by #129. Creating new issue for doc tasks
